File: festival-kotlin-web/src/main/java/cn/bdqfork/web/route/response/TextPlainResponseHandler.java
package cn.bdqfork.web.route.response;


import io.vertx.core.http.HttpServerResponse;

/**
 * @author bdq
 * @since 2020/2/12
 */
public class TextPlainResponseHandler extends AbstractResponseHandler {

    @Override
    protected void doHandle(HttpServerResponse response, Object result) throws Exception {
        response.end(result.toString());
    }
}

File: festival-kotlin-web/src/main/kotlin/cn/bdqfork/kotlin/web/route/response/TextPlainResponseHandler.kt
package cn.bdqfork.kotlin.web.route.response

import io.vertx.ext.web.RoutingContext

/**
 * @author bdq
 * @since 2020/2/12
 */
class TextPlainResponseHandler : AbstractResponseHandler() {
    @Throws(Exception::class)
    override fun doHandle(routingContext: RoutingContext, result: Any) {
        routingContext.response().end(result.toString())
    }
}